optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/SolverBenchmarkConfig.java

```diff
@@ -20,16 +20,20 @@
 import java.util.regex.Pattern;
 
 import com.thoughtworks.xstream.annotations.XStreamAlias;
+import org.apache.commons.lang3.ObjectUtils;
 import org.optaplanner.benchmark.impl.result.PlannerBenchmarkResult;
 import org.optaplanner.benchmark.impl.result.SingleBenchmarkResult;
 import org.optaplanner.benchmark.impl.result.SolverBenchmarkResult;
 import org.optaplanner.core.config.solver.SolverConfig;
+import org.optaplanner.core.config.util.ConfigUtils;
 
 @XStreamAlias("solverBenchmark")
 public class SolverBenchmarkConfig {
 
     private String name = null;
 
+    private Integer subSingleCount = null;
```

Comment:
IIRC, the order should be solverConfig, problemBenchmarksConfig, subSingleCount because users first think about the others before deciding how many subsingles to run